src/tests/lxc-test-config-jump-table

```diff
@@ -0,0 +1,228 @@
+#! /bin/bash
```

Comment:
Sorry, these two file should be added to `.gitignore`, see https://github.com/lxc/lxc/pull/1640